f8a_worker/dispatcher/flows/stackApiGraphV2Flow.yml

```diff
@@ -12,3 +12,4 @@
           to:
             - 'recommendation_v2'
             - 'stack_aggregator_v2'
+            - 'BookkeeperTask'
```

Comment:
We would like to keep edges as minimal as possible (see http://selinon.readthedocs.io/en/latest/practices.html#make-flow-edges-as-minimal-as-possible ), so it would be better to rewrite this flow definition to:

```yaml

        - from:
          to: 'GraphAggregatorTask'
        - from:
          to: 'ManifestKeeperTask'
        - from: 'GraphAggregatorTask'
          to: 'recommendation_v2'
        - from: 'GraphAggregatorTask'
          to: 'stack_aggregator_v2'
        - from: 'GraphAggregatorTask'
          to: 'BookkeeperTask'
```